src/Thruway/Role/Subscriber.php

```diff
@@ -128,6 +134,37 @@ protected function processSubscribed(ClientSession $session, SubscribedMessage $
             }
         }
     }
+    
+    /**
+     * Process unsubscribe error
+     * 
+     * @param \Thruway\AbstractSession $session
+     * @param \Thruway\Message\ErrorMessage $msg
+     */
+    protected function processUnsubscribeError(AbstractSession $session, ErrorMessage $msg)
```

Comment:
This method is called whenever the router responds with an `ERROR` message. Why should it be called `processUnsubscribe`? It follows the same logic as `processSubscribeError`.